pkgs/os-specific/linux/firejail/default.nix

```diff
@@ -27,7 +27,7 @@ stdenv.mkDerivation {
   '';
 
   preBuild = ''
-    sed -e "s@/etc/@$out/etc/@g" -i Makefile
+    sed -e "s@/etc/@$out/etc/@g" -e "/chmod u+s/d" -i Makefile
```

Comment:
This change deletes the entire `chmod u+s` invocation entirely, which seems even better! (yes?)